### DISCUSSION TITLE
Updating Release Date with Version Bump

### DISCUSSION THREAD

**Discussion by SterlingPeet** [callowayproject/bump-my-version#141] [Q&A] [answered]
Is there a reasonable way to update the dates that are located alongside version numbers in my code, from ``bump-my-version``?  For example, I have an ``__init__.py`` that looks like this (but there are other places, like in the docs and readme, too):

```
__date__ = '2022-12-19'
__version__ = '0.4.0'
```

And I would like for it update to this if I run ``bump-my-version bump minor`` today (Feb 23, 2024):

```
__date__ = '2024-02-23'
__version__ = '0.5.0'
```

As of today, my config would just update that version number and I have to go back through and fix the dates before tagging.  Great tool btw, its already saved me a ton of time and headache.

**Comment by coordt** [comment, marked as answer]
Great question! As such, I have turned the question and answer into a "How-to" in the docs: [How to update a date](https://callowayproject.github.io/bump-my-version/howtos/update-a-date.html)
